hazelcast-jet-all/src/main/java/com/hazelcast/jet/server/JetCommandLine.java

```diff
@@ -712,4 +813,373 @@ static Throwable peel(Throwable e) {
+        AttributedStringBuilder builder = new AttributedStringBuilder()
+                .style(AttributedStyle.BOLD.foreground(SECONDARY_COLOR));
+        builder.append("|");
+        int colSize = row.getMetadata().getColumnCount();
+        for (int i = 0; i < colSize; i++) {
+            String colValue = row.getObject(i).toString();
+            centralize(colWidth, builder, colValue);
+        }
+        out.println(builder.toAnsi());
+        out.flush();
+    }
+
+    private static void centralize(int colWidth, AttributedStringBuilder builder, String colValue) {
+        builder.style(AttributedStyle.DEFAULT.foreground(PRIMARY_COLOR));
+        int wsLen = 0;
+        if (colValue.length() < colWidth) {
+            wsLen = colWidth - colValue.length();
+        } else {
+            colValue = colValue.substring(0, colWidth - 3) + "...";
+        }
+
+        for (int j = 0; j < wsLen / 2; j++) {
+            builder.append(' ');
+        }
+        builder.append(colValue);
+        for (int j = 0; j < wsLen / 2 + wsLen % 2; j++) {
+            builder.append(' ');
+        }
+        builder.style(AttributedStyle.BOLD.foreground(SECONDARY_COLOR));
+        builder.append('|');
+    }
+
+    private static void printSeparatorLine(int colSize, int colWidth, PrintWriter out) {
+        AttributedStringBuilder builder = new AttributedStringBuilder()
+                .style(AttributedStyle.BOLD.foreground(SECONDARY_COLOR));
+        builder.append('+');
+        for (int i = 0; i < colSize; i++) {
+            for (int j = 0; j < colWidth; j++) {
+                builder.append('-');
+            }
+            builder.append('+');
+        }
+        out.println(builder.toAnsi());
+    }
+
+    private static class CliPrompts {
+        static final String STARTING_PROMPT = new AttributedStringBuilder()
+                .style(AttributedStyle.BOLD.foreground(SECONDARY_COLOR))
+                .append(
+                "\to   o   o   o---o o---o o     o---o   o   o---o o-o-o        o o---o o-o-o   o---o   o---o   o\n" +
+                "\t|   |  / \\     /  |     |     |      / \\  |       |          | |       |     |       |   |   |\n" +
+                "\to---o o---o   o   o-o   |     o     o---o o---o   |          | o-o     |     o---o   o   o   o\n" +
+                "\t|   | |   |  /    |     |     |     |   |     |   |      \\   | |       |         |   |   |   |\n" +
+                "\to   o o   o o---o o---o o---o o---o o   o o---o   o       o--o o---o   o     o---o   o---\\\\  o---")
+                .style(AttributedStyle.BOLD.foreground(PRIMARY_COLOR))
+                .append("\n\t\t\t\t Welcome to the Hazelcast Jet SQL Console(BETA)")
+                .append("\n\t\t\t\t Commands end with semicolon")
+                .append("\n\t\t\t\t Type 'HELP;' to display the available commands")
+                .append("\n\t\t\t\t Press Ctrl+C to cancel streaming queries\n\n")
```

Comment:
I literally converted it to what you want. But when this console is connected to an imdg cluster, It should show "Connected to Hazelcast IMDG 4.1" kind of prompt but I have not done it yet.

Reply:
I am actually unsure if a JetClientInstance can connect to an IMDG cluster. Please check that.

Reply:
It can connect to an IMDG cluster. `./jet -v sql --targets dev@localhost:5701`(dev -> default cluster name for IMDG) this command can be used to connect IMDG cluster running on localhost:5701. In the starting prompt, the sql console indicate which type of cluster (Jet or IMDG) the client connected to. I will also update the docs link in the help command to show the imdg documentation in case the client is connected to imdg.